addons/addon-base/packages/services/lib/authorization/authorization-utils.js

```diff
@@ -35,6 +35,10 @@ function isRoot(requestContext) {
   return _.get(requestContext, 'principal.userType', '') === 'root';
 }
 
+function isSystem(requestContext) {
```

Comment:
Just a heads up (not PR blocking): This method already exists on `addons/addon-base-raas/packages/base-raas-services/lib/helpers/is-role.js`